src/ui/components/connect/listConnections/listConnections.html

```diff
@@ -4,7 +4,7 @@
   </div>
   <ul class="connection-list">
     <li ng-repeat="connection in connections" ng-class="{ 'active' : connection.selected }" ng-show="connections.length">
-      <a class="name" href="#" ng-click="selectConnection(connection, $event)">
+      <a class="name" href="#" ng-click="selectConnection(connection, $event)" ng-dblclick="selectConnection(connection, $event);connect($event)">
```

Comment:
I'm open to just only having double click open up a new query tab. Single click could change to just select the current collection in the sidebar, and double-click would open it. What do you think?